avm/res/web/site/tests/e2e/appLinux.defaults/main.test.bicep

```
@@ -0,0 +1,60 @@
+targetScope = 'subscription'
+
+metadata name = 'Linux Web App, using only defaults'
+metadata description = 'This instance deploys the module as a Linux Web App with the minimum set of required parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-web.sites-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'wswamin'
```

Careful, this will case to deployments to overlap as the serviceShort is used also by the `webAppMin` test.

I'd suggest
```suggestion
param serviceShort string = 'wswalmin'
```
This may require a refresh of the readme

---

Thanks Alex, good spot. :)